builder/comp-builder.nix

```diff
@@ -180,10 +182,9 @@ let
       ++ lib.optionals useLLVM [
         "--ghc-option=-fPIC" "--gcc-option=-fPIC"
         ]
+      ++ map (o: ''--ghc${lib.optionalString (stdenv.hostPlatform.isGhcjs) "js"}-options="${o}"'') ghcOptions
```

Comment:
Just to be clear, the idea is that by including them in the call to `configure` we don't need to include them everywhere else like we were doing before?

Reply:
Yes I noticed that we passed other `--ghc-options` arguments (like the `enableDWARF` one) to configure instead of to build.  I can't think of a reason to treat them differently.  It was confusing that the list of configure args that we log to the output in the did not include the `--ghc-options` from this source.

Reply:
Yeah, it seems way better if it works!